### ISSUE TITLE
SplitButton CornerRadius

### ISSUE THREAD

**Issue by amkuchta** [MahApps/MahApps.Metro#2871]
## What steps will reproduce this issue?

_Creating a custom-styled `SplitButton` and applying `CornerRadius` does not have any effect:_

```xaml
<Style x:Key="SplitButtonStyle" TargetType="{x:Type Controls:SplitButton}">
  <Setter Property="Foreground" Value="{DynamicResource TextBrush}" />
  <Setter Property="Background" Value="{DynamicResource GrayBrush10}" />
  <Setter Property="BorderBrush" Value="{DynamicResource TextBoxBorderBrush}" />
  <Setter Property="BorderThickness" Value="1" />
  <Setter Property="Controls:ControlsHelper.ContentCharacterCasing" Value="Upper" />
  <Setter Property="Controls:ControlsHelper.CornerRadius" Value="3" />
  <Setter Property="FontFamily" Value="{DynamicResource DefaultFont}" />
  <Setter Property="FontSize" Value="{DynamicResource UpperCaseContentFontSize}" />
  <Setter Property="FontWeight" Value="Bold" />
  <Setter Property="MinHeight" Value="25" />
  <Setter Property="Padding" Value="2 3" />
</Style>

...

<Controls:SplitButton Style="{StaticResource SplitButtonStyle}" />
```

### Expected outcome

_`CornerRadius` should be set for all corners to a value of "3"_

### Repo

_https://github.com/amkuchta/Vulnerator/tree/code_cleanup_

### Environment

- MahApps.Metro __v1.5.0.0__
- Windows OS __10__
- Visual Studio __2015__
- .NET Framework __4.5.2__

**Comment by punker76** [comment]
This will be available in 1.5.0

**Comment by amkuchta** [comment]
@punker76 Thanks!  Did this make the cut for `1.5.0_alpha006`, or do I need to wait for `007`?  Just updated to `006`, and it still isn't working, which is why I am asking.
